src/schemas/epicPayload.schema.json

```diff
@@ -13,37 +13,17 @@
                 "platformId": {
                     "type": "string"
                 },
-                "campaignCode": {
-                    "type": "string"
-                },
-                "abTestName": {
-                    "type": "string"
-                },
-                "abTestVariant": {
-                    "type": "string"
-                },
                 "referrerUrl": {
                     "type": "string"
                 }
             },
             "required": [
-                "abTestName",
-                "abTestVariant",
-                "campaignCode",
                 "ophanComponentId",
                 "ophanPageId",
                 "platformId",
                 "referrerUrl"
             ]
         },
-        "localisation": {
-            "type": "object",
-            "properties": {
-                "countryCode": {
-                    "type": "string"
-                }
-            }
-        },
         "targeting": {
             "type": "object",
             "properties": {
```

Comment:
Should we not expect to see `countryCode` as a property of `targeting` now? 🤔

Reply:
Oh good point, that sounds right. I'll check...

Reply:
Oh ha looks like we already had it - so the only change here is to remove the separate localisation object.